src/graphics/transformation.rs

```diff
@@ -63,6 +63,26 @@ impl Transformation {
     pub fn rotate(rotation: f32) -> Transformation {
         Transformation(Matrix3::new_rotation(rotation))
     }
+
+    /// Projects a point via this transformation.
+    ///
+    /// Can be used to convert screen-coordinates to world-coordinates.
+    pub fn project(&self, point: Point) -> Option<Point> {
+        let inverse_mat = self.0.try_inverse()?;
+
+        let projected = inverse_mat * Vector3::new(point.x, point.y, 1.0);
```

Comment:
Looks like we need to bring `Vector3` into scope.

Reply:
Ah, whoops, forgot to copy-paste that part 😅

Reply:
We are missing `Point` now :sweat_smile:

Reply:
Ah geez, I'd try to test myself but for some reason in my cloned fork only I keep getting this compile error.

```
 Compiling wgpu_glyph v0.3.1 (https://github.com/hecrj/wgpu_glyph?rev=0577e4d2be6b035a14aa0c5d82b143aaf26c1bd3#0577e4d2)
error: there is no argument named `old`
   --> C:\Users\John\.cargo\git\checkouts\wgpu_glyph-12f1f20822f22da3\0577e4d\src\lib.rs:157:60
    |
157 | ...                   "Increasing glyph texture size {old:?} -> {new:?}. \
    |                                                      ^^^^^^^

error: there is no argument named `new`
   --> C:\Users\John\.cargo\git\checkouts\wgpu_glyph-12f1f20822f22da3\0577e4d\src\lib.rs:157:71
    |
157 | ...                   "Increasing glyph texture size {old:?} -> {new:?}. \
    |                                                                 ^^^^^^^

error: there is no argument named `new`
   --> C:\Users\John\.cargo\git\checkouts\wgpu_glyph-12f1f20822f22da3\0577e4d\src\lib.rs:158:42
    |
158 | ...                   Consider building with `.initial_cache_size({new:?})` to avoid \
    |                                   ^^^^^^^

error: multiple unused formatting arguments
   --> C:\Users\John\.cargo\git\checkouts\wgpu_glyph-12f1f20822f22da3\0577e4d\src\lib.rs:160:29
    |
157 | / ...                   "Increasing glyph texture size {old:?} -> {new:?}. \
158 | | ...                    Consider building with `.initial_cache_size({new:?})` to avoid \
159 | | ...                    resizing",
    | |________________________________- multiple missing formatting specifiers
160 |   ...                   old = self.glyph_brush.texture_dimensions(),
    |                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ argument never used
161 |   ...                   new = (new_width, new_height),
    |                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ argument never used

error[E0425]: cannot find value `old` in this scope
   --> C:\Users\John\.cargo\git\checkouts\wgpu_glyph-12f1f20822f22da3\0577e4d\src\lib.rs:160:29
    |
160 | ...                   old = self.glyph_brush.texture_dimensions(),
    |                       ^^^ not found in this scope

error[E0425]: cannot find value `new` in this scope
   --> C:\Users\John\.cargo\git\checkouts\wgpu_glyph-12f1f20822f22da3\0577e4d\src\lib.rs:161:29
    |
161 | ...                   new = (new_width, new_height),
    |                       ^^^ not found in this scope

error: aborting due to 6 previous errors

For more information about this error, try `rustc --explain E0425`.
error: could not compile `wgpu_glyph`.
```

Reply:
You should be able to fix that with `cargo update`. The `log` folks have yanked the broken version.

Reply:
Ah I see now that's what you meant earlier by the log crate being broken. It works now!